admin/acceptance.py

```diff
@@ -156,23 +127,26 @@ def __init__(self):
             raise UsageError("Unsupored varianta: %s."
                              % (', '.join(self.variants),))
 
-    def start_nodes(self):
+    @inlineCallbacks
```

Comment:
Recent discussion at https://zulip.com/#narrow/stream/engineering/subject/inlineCallbacks concluded with a decision to not use inlineCallbacks in acceptance tests.